### DISCUSSION TITLE
Request: Additional assignee to a task.

### DISCUSSION THREAD

**Discussion by aquatic7** [kanboard/kanboard#5433] [Ideas]
There's a real need to be able to assign additional assignee to a task.
Projects are often not just run by one person. There might be more than one. 
Like ServiceNow do this, is by an assignee and then the ability to add additional assignee.
It would be so useful to manage tasks as a team, where several people are part of a task.

**Comment by creecros** [comment]
There is a plugin for that. 
https://github.com/creecros/Group_assign

**Reply by jifoxpa** [reply]
Is anyone still maintaining this plugin? This plugin is not useable anymore.

See: https://github.com/creecros/Group_assign/issues
